dplace_app/api_views.py

```diff
@@ -222,7 +222,7 @@ def join_values(varid, criteria):
             elif operator == 'gt':
                 sql_where.append("{0}.coded_value_float >= {1:f}".format(alias, params[0]))
             elif operator == 'lt':
-                sql_where.append("{0}.coded_value_float <= {1:f}".format(alias, params[0]))
+                sql_where.append("{0}.coded_value_float <= {1:f}".format(alias, params[1]))
```

Comment:
@Bibiko this should probably be updated for the actual website straight away, because it isn't working on d-place.org at the moment.